backend/run.py

```diff
@@ -0,0 +1,5 @@
+from manage import app
+import views.views
```

Comment:
I think this import views.views not need.
it nowhere used in code

Reply:
this import is needed